purchases/src/main/kotlin/com/revenuecat/purchases/common/offerings/OfferingsManager.kt

```diff
@@ -28,19 +37,54 @@ internal class OfferingsManager(
         onSuccess: ((Offerings) -> Unit)? = null,
         fetchCurrent: Boolean = false,
     ) {
+        trackGetOfferingsStartedIfNeeded()
+        val startTime = dateProvider.now
+        val onErrorWithTracking: (PurchasesError, DiagnosticsTracker.CacheStatus) -> Unit = { error, cacheStatus ->
+            trackGetOfferingsResultIfNeeded(startTime, cacheStatus, error, null, null)
+            onError?.invoke(error)
+        }
+        val onSuccessWithTracking: (OfferingsResultData, DiagnosticsTracker.CacheStatus) -> Unit =
+            { result, cacheStatus ->
+                trackGetOfferingsResultIfNeeded(
+                    startTime,
+                    DiagnosticsTracker.CacheStatus.NOT_CHECKED,
```

Comment:
I think this should be 
```suggestion
                    cacheStatus,
```

Reply:
Right! Thanks for catching! It was a last-minute refactor to please `detekt` and I totally missed it. Thanks again! 🙏